experimental/framework/frs-client/package.json

```diff
@@ -14,11 +14,13 @@
     "build": "npm run build:genver && concurrently npm:build:compile npm:lint",
     "build:commonjs": "npm run tsc && npm run build:test",
     "build:compile": "concurrently npm:build:commonjs npm:build:esnext",
+    "build:docs": "api-extractor run --local && copyfiles -u 1 ./_api-extractor-temp/doc-models/* ../../../_api-extractor-temp/",
```

Comment:
looks like it's pretty inconsistent throughout the codebase, but api-extractor should probably be under devDependencies for everything that uses it

Reply:
Oooh, good point. I'll fix that for these 4 at least. In the future npm7 and/or yarn2 will help prevent this.